internal/models/kubernetessecret/cluster/secret.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package secret
+
+import (
+	"github.com/go-openapi/swag"
+	objectmetamodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/objectmeta"
+)
+
+// VmwareTanzuManageV1alpha1ClusterNamespaceSecretSecret Represents Tanzu Secret.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.namespace.secret.Secret
+type VmwareTanzuManageV1alpha1ClusterNamespaceSecretSecret struct {
+
+	// Full name for the Secret.
+	FullName *VmwareTanzuManageV1alpha1ClusterNamespaceSecretFullName `json:"fullName,omitempty"`
+
+	// Metadata for the Secret  object.
+	Meta *objectmetamodel.VmwareTanzuCoreV1alpha1ObjectMeta `json:"meta,omitempty"`
+
+	// Spec for the Secret.
+	Spec *VmwareTanzuManageV1alpha1ClusterNamespaceSecretSpec `json:"spec,omitempty"`
+
+	// Status for the Secret.
+	Status *VmwareTanzuManageV1alpha1ClusterNamespaceStatus `json:"status,omitempty"`
```

Comment:
If it is not something specific to secrets can we move it to a common folder/package and import it to use here?

Reply:
Updated PTAL